ArduCopter/mode_althold.cpp

```diff
@@ -131,8 +134,8 @@ void Copter::ModeAltHold::run()
 #else
         attitude_control->reset_rate_controller_I_terms();
         attitude_control->set_yaw_target_to_current_heading();
+        attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(target_roll, target_pitch, target_yaw_rate);
 #endif
-        attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(target_roll, target_pitch, target_yaw_rate, get_smoothing_gain());
```

Comment:
Is this really intended to be removed from executing in Heli? Asking because this is important for the discussion in #7467 (see all previous PRs as well) - specifically, it looked like Heli people would want Heli to be controllable even when landed but maybe Multi didn't; this changes it for the opposite, which makes it more confusing for me  🙂

Reply:
This is definitely a bug.  I will fix in a follow-up commit, thanks!